2.4/root/usr/bin/run-mongod

```diff
@@ -28,9 +25,6 @@ function unset_env_vars() {
 }
 
 function cleanup() {
-  if [ -n "${MONGODB_REPLICA_NAME-}" ]; then
-    mongo_remove
-  fi
```

Comment:
@omron93 Correct me if I'm wrong that this script is always gets executed for standalone mongodb installation. That's why I'm removing the code that is useful only for replicaset.